### PR TITLE
Adding external ssl support for gcp and aws

### PR DESCRIPTION
**Pull request by juandiegopalomino**
The trickiest was GCP. You see, a gcp http load balancer will only forward 443 requests if there is a gcp ssl attached to it, which suck. I considered provisioning an L4 TCP load balancer just for this case (we don't use it before because it does not handle ssl certs at all-- so with the gcp lbs ssl is either required for port 443 or impossible to handle) and I thought it's way too much of a complication. So in GCP we upload the certificate to GCP, and then also add it to the nginx ingress as the default cert just to keep that behavior consistent thoughout all 3 clouds.

### CODE REVIEW

File: config/tf_modules/k8s-service/k8s-service/templates/ingress.yaml [hunk 1]
```diff
@@ -24,14 +24,6 @@ metadata:
     nginx.ingress.kubernetes.io/proxy-send-timeout: "3600"
     {{- end }}
 spec:
-  tls:
```
Comment (legalosLOTR) [inline config/tf_modules/k8s-service/k8s-service/templates/ingress.yaml:27]: Would this have any backwards compatibility consequences?

File: examples/airflow/README.md [hunk 2]
```diff
@@ -1,12 +0,0 @@
-# Description
-
```
Comment (legalosLOTR) [inline examples/airflow/README.md:2]: Why is this being deleted?

File: config/tf_modules/k8s-service/k8s-service/templates/ingress.yaml (hunk 1 above)
Comment (juandiegopalomino) [reply config/tf_modules/k8s-service/k8s-service/templates/ingress.yaml:27]: No

File: examples/airflow/README.md (hunk 2 above)
Comment (juandiegopalomino) [reply examples/airflow/README.md:2]: I thought this was a typo/incorrectly added file. Do you want to keep it?

Comment (legalosLOTR) [reply examples/airflow/README.md:2]: Yup we should keep these files

**Review by legalosLOTR — APPROVED** [review]

**Pull request merged**